upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -639,286 +694,295 @@ data:
   "batchMaxDuration": "10s"
   "batchIdleDuration": "1s"
 ---
-# Source: karpenter/templates/controller/rbac.yaml
+# Source: karpenter/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: karpenter-global-settings
+  namespace: kube-system
+data:  
+    "aws.clusterEndpoint": "https://{{ APIInternalName }}"
+    "aws.clusterName": "{{ ClusterName }}"
+    "aws.defaultInstanceProfile": ""
+    {{ if not .Networking.AmazonVPC }}
+    "aws.enableENILimitedPodDensity": "true"
+    {{ else }}
+    "aws.enableENILimitedPodDensity": "false"
+    {{ end }}
+    "aws.enablePodENI": "false"
+    "aws.interruptionQueueName": "{{ ClusterName }}"
```

Comment:
Do we need to set this? I would like to keep the NTH functionality out of Karpenter on kOps and point people to using the NTH addon instead.

Reply:
Let disable it then :+1: 
 - https://github.com/kubernetes/kops/pull/15144/commits/ba5384b6a1320d7e93027be73106554ad9b935c9